5.0 refactor mqtt specific (#1555)
emqtt/emq-docs-cn: en_US/mqtt/mqtt-auto-subscription.md

> # Auto Subscription

Author: 名字就是这个 使用名词
```suggestion
# Auto Subscribe
```

With **Auto Subscription** enabled, users can set multiple EMQX rules. After a client is successfully connected to EMQX, EMQX will complete the subscription process for the client automatically, and the clients no longer need to send `SUBSCRIBE` requests. 

> Before EMQX5.0, this feature is called **Proxy Subscription**.

Author: ```suggestion
Before EMQX 5.0, this feature is called **Proxy Subscription**.
```

}
```

```bash
+---------------------------+             +----------------+
| clientid: demo_client1    |             |  EMQX Broker   |

> | u-----------------------n | u---------- |  u----------   |

Author: ```suggestion
| username: admin | | |
```
